Automattic/jetpack
# require-reviews: Make it a project to publish

#### Changes proposed in this Pull Request:

This will take over the [Automattic/action-required-review](https://github.com/automattic/action-required-review) repo with the updated version in our monorepo.

#### Jetpack product discussion


We've talked about it, but I can't find a good link.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Does the required review check still show up?
* Check the build artifact to see if the content to be mirrored to the repo looks sane.

#### Proposed changelog entry for your changes:


* None needed.
